src/Stripe.net/Services/Accounts/AccountUpdateOptions.cs

```diff
@@ -25,6 +25,18 @@ public class AccountUpdateOptions : BaseOptions, IHasMetadata
         [JsonProperty("business_type")]
         public string BusinessType { get; set; }
 
+        /// <summary>
+        /// Each key of the dictionary represents a capability,
+        /// and each capability maps to its settings (e.g. whether
+        /// it has been requested or not). Each capability will be
+        /// inactive until you have provided its specific
+        /// requirements and Stripe has verified them. An account
+        /// may have some of its requested capabilities be active
+        /// and some be inactive.
+        /// </summary>
+        [JsonProperty("capabilities")]
+        public AccountCapabilitiesOptions Capabilities { get; set; }
+
         /// <summary>
```

Comment:
In theory you should make `RequestedCapabilities` obsolete now

Reply:
done